Projects/Food_Website/style.css

```diff
@@ -24,6 +24,7 @@ body{
 }
 .title{
     padding: 0 30px;
+    padding-top: 29px;
```

Comment:
Why the changes made in FOOD_Website is present here??
Please delete this commit , the other changes are well made , So that I can merge this PR